python/cudf/cudf/_libxx/reduce.pyx

```diff
@@ -0,0 +1,67 @@
+from cudf._libxx.cpp.reduce cimport cpp_reduce, cpp_scan, scan_type
+from cudf._libxx.cpp.scalar.scalar cimport scalar
+from cudf._libxx.cpp.types cimport data_type, type_id
+from cudf._libxx.cpp.column.column_view cimport column_view
+from cudf._libxx.cpp.column.column cimport column
+from cudf._libxx.scalar cimport Scalar
+from cudf._libxx.column cimport Column
+from cudf._libxx.types import np_to_cudf_types
+from cudf._libxx.move cimport move
+from cudf._libxx.aggregation cimport get_aggregation, aggregation
+from libcpp.memory cimport unique_ptr
+import numpy as np
+
+cpdef reduce(reduction_op, Column incol, kwargs=None, dtype=None, ddof=1):
```

Comment:
Any reason not to make this a `def` and then use `**kwargs` instead of an explicit `kwargs`?